common/components/BalanceSidebar/TokenBalances/AddCustomTokenForm/AddressField.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Input } from 'components/ui';
+import { translateRaw } from 'translations';
+import { IGenerateAddressLookup } from './AddCustomTokenForm';
+import { isValidETHAddress } from 'libs/validators';
+import { Result } from 'nano-result';
+
+interface OwnProps {
+  addressLookup: IGenerateAddressLookup;
+  onChange(address: Result<string>): void;
+}
+
+enum ErrType {
```

Comment:
When should these go in translations vs. inside a component? Seems counter to internationalization.

Reply:
The previous component that this one is based off had inline errors, so i just kept as-is